spec/support/vcr_cassettes/sm_client/messages/creates/cannot_send_draft_as_reply.yml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
I don't think we should need a yml file for these, these should be tested as validations and validations only.